a10_octavia/common/a10constants.py

```diff
@@ -93,6 +93,7 @@
 CONFIRM_VRRP_STATUS = 'confirm-vrrp-status'
 ALLOCATE_VIP = 'allocate-vip'
 UPDATE_VIP_AFTER_ALLOCATION = 'update-vip-after-allocation'
+GET_COMPUTE_FOR_PROJECT = 'get_compute_for_project'
```

Comment:
Need to turn underscores '_' to dashes '-'

Reply:
done